### PR TITLE
feat(gitlab-tags): Add support for "releaseTimestamp" for GitLab tags

### PR DESCRIPTION
**Pull request by zharinov**
Ref #4689

### CODE REVIEW

File: lib/datasource/gitlab-tags/index.ts [hunk 1]
```diff
@@ -36,32 +43,30 @@ export async function getPkgReleases({
...
     sourceUrl: `${depHost}/${repo}`,
     releases: null,
   };
-  dependency.releases = versions.map(version => ({
-    version,
-    gitRef: version,
+  dependency.releases = gitlabTags.map(({ name, commit }) => ({
+    version: name,
+    gitRef: name,
+    releaseTimestamp: commit.created_at,
```
Comment (rarkins) [inline lib/datasource/gitlab-tags/index.ts]: Is it definitely a mandatory field that's always present?

File: lib/datasource/gitlab-tags/index.spec.ts [hunk 2]
```diff
@@ -14,7 +14,20 @@ describe('datasource/gitlab-tags', () => {
...
+          name: 'v1.0.0',
+          commit: {
+            created_at: '2020-03-04T12:01:37.000-06:00',
+          },
+        },
+        {
+          name: 'v1.1.0',
+          commit: {
+            created_at: '2020-03-04T12:12:13.000-06:00',
+          },
```
Comment (viceice) [inline lib/datasource/gitlab-tags/index.spec.ts]:
```suggestion
          commit: { },
```

for null testing

File: lib/datasource/gitlab-tags/__snapshots__/index.spec.ts.snap [hunk 3]
```diff
@@ -5,10 +5,12 @@ Object {
   "releases": Array [
     Object {
       "gitRef": "v1.0.0",
+      "releaseTimestamp": "2020-03-04T12:01:37.000-06:00",
       "version": "v1.0.0",
     },
     Object {
       "gitRef": "v1.1.0",
+      "releaseTimestamp": "2020-03-04T12:12:13.000-06:00",
```
Comment (viceice) [inline lib/datasource/gitlab-tags/__snapshots__/index.spec.ts.snap]:
```suggestion
      "releaseTimestamp": undefined,
```

File: lib/datasource/gitlab-tags/index.ts [hunk 4]
```diff
@@ -36,32 +43,30 @@ export async function getPkgReleases({
...
     sourceUrl: `${depHost}/${repo}`,
     releases: null,
   };
-  dependency.releases = versions.map(version => ({
-    version,
-    gitRef: version,
+  dependency.releases = gitlabTags.map(({ name, commit }) => ({
+    version: name,
+    gitRef: name,
+    releaseTimestamp: commit && commit.created_at ? commit.created_at : null,
```
Comment (viceice) [inline lib/datasource/gitlab-tags/index.ts]:
```suggestion
    releaseTimestamp: commit?.created_at,
```

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/datasource/gitlab-tags/index.ts (hunk 4 above)
Comment (zharinov) [reply lib/datasource/gitlab-tags/index.ts]: Love this feature so much!

**Pull request merged**